portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/contact-me")
+public class ContactServlet extends HttpServlet {
+
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // Gets input from the contact-input form
+        String name = getParameter(request, "name", "");
+        String phone = getParameter(request, "number", "");
+        String email = getParameter(request, "email", "");
+        ArrayList<Person> contactList = new ArrayList<Person>();
+        
+        Person newContact = new Person(name, phone, email);
+
+        // Temp. Storage for information
+        // In the future, add a file or something of the sort to store contacts
```

Comment:
A small suggestion: Usually these comments start with some kind of prefix, like "TODO(tris): In the future, add a file or something of the sort [...]". That way, you can find all of them in your codebase quickly by searching for the string TODO.

Obviously it's less important when there's only 1 or 2 things that you want to fix, but you may find it helpful when doing work in a larger group to keep better track of TODOs.